src/pages/iou/MoneyRequestTagPage.js

```diff
@@ -109,6 +109,14 @@ export default compose(
             key: ONYXKEYS.IOU,
         },
     }),
+    //  eslint-disable-next-line
+    withOnyx({
```

Comment:
I don't think this PR should be adding any new uses of `withOnyx`. Was it a mistake to include this?

Reply:
Yess, this was a leftover that I added again, it's deleted now @tgolen